pages/starsearch.tsx

```diff
@@ -129,25 +206,35 @@ function Chatbox({ author, content, userId }: { author: "You" | "StarSearch"; co
   return (
     <li className="flex gap-2 items-start my-4">
       {renderAvatar()}
-      <Card className="flex flex-col grow bg-white z-10 p-2 lg:p-4">
+      <Card className="flex flex-col grow bg-white z-10 p-2 lg:p-4 lg:max-w-5xl">
         <h3 className="font-semibold text-sauced-orange">{author}</h3>
         <p>{content}</p>
       </Card>
     </li>
   );
 }
 
-function StarSearchInput({ onSubmit }: { onSubmit: () => void }) {
+function StarSearchInput({
+  isRunning,
+  onSubmitPrompt,
+}: {
+  isRunning: boolean;
+  onSubmitPrompt: (prompt: string) => void;
+}) {
+  const [promptInput, setPromptInput] = useState<string>("");
   return (
     <section className="w-full h-fit max-w-4xl px-1 py-[3px] rounded-xl bg-gradient-to-r from-sauced-orange via-amber-400 to-sauced-orange">
       <div className="w-full h-fit flex justify-between rounded-lg">
```

Comment:
Refactor (blocking): This should be a form. Right now the required field and other things you get from a form are not available because it's all wrapped in a form. I'll put some notes shortly.